server/src/main/java/org/elasticsearch/action/admin/indices/recovery/RecoveryResponse.java

```diff
@@ -39,7 +39,6 @@
  */
 public class RecoveryResponse extends BroadcastResponse implements ToXContentFragment {
 
-    private boolean detailed = false;
```

Comment:
wasn't this already removed in another PR? Maybe you need to merge master in?

Reply:
Yes, I didn't merge master before.